kernel/poll.c

```diff
@@ -230,7 +230,7 @@ static inline int register_events(struct k_poll_event *events,
 			register_event(&events[ii], poller);
 			events_registered += 1;
 		} else {
-			;
+			; /* Non-empty final else in if ... else if struct */
```

Comment:
Hi @andyross , do you think either of these comments are ok to describe this final else here?

`No action needed. Event is not of interest to register.`
or 
`Event is not one of those identified in is_condition_met() catching non-polling events, or the event is marked for just check, or not marked for polling.`

Reply:
IMHO it's cleaner to structure it like:

```c
if(is_condition_met(...)) {
    ....
} else {
    if(whatever) {
        ...
    }
}
```

Basically if there's a straightforward translation that preserves the meaning and doesn't require the weird empty expression, choose that. :)

Reply:
Hi @andyross yes I agree, but we missed that opportunity in my earlier PRs. So at this time, we need to adjust the comments please. I am now tasked with correcting the comments for the structures in place, and code owners can refactor as enhancement later. 

That being said, do you think either of the comments I drafted are OK or can you please suggest a better one?